src/osemgrep/networking/Rule_fetching.mli

```diff
@@ -97,12 +99,12 @@ val load_rules_from_file :
   registry_caching:bool ->
   < Cap.network > ->
   Fpath.t ->
-  rules_and_origin
+  (rules_and_origin, Rule.error) Either.t
```

Comment:
I think for this it's better to use the Result.t type (with Ok and Error),
It was designed for that.